src/vs/base/common/actionWidget/actionWidget.ts

```diff
@@ -0,0 +1,19 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IDisposable } from 'vs/base/common/lifecycle';
+
+export interface ActionSet<T> extends IDisposable {
```

Comment:
If this is all that's in base and isn't used anywhere, it should go to `platform/actionWidget/common`

Reply:
I put it there so it can be referenced in /Users/meganrogge/Repos/vscode/src/vs/editor/contrib/codeAction/common/types.ts

Reply:
oh will try it in common